packages/core/src/shared/utilities/messages.ts

```diff
@@ -75,7 +86,16 @@ export async function showMessageWithUrl(
     const uri = typeof url === 'string' ? vscode.Uri.parse(url) : url
     const items = [...extraItems, urlItem]
 
-    const p = showMessageWithItems(message, kind, items, useModal)
+    const p = showMessage(
+        kind,
+        message,
+        items,
+        { modal: useModal },
+        {
+            id: 'showMessageWithUrl',
+            reasonDesc: getTelemetryReasonDesc(message),
```

Comment:
Setting `reasonDesc` based on `message`. Maybe in the future, `showMessageWithUrl` should allow the caller to set `id`.